### ISSUE TITLE
Disallow Future date as birth date on sign-up page 

### ISSUE THREAD

**Issue by mahak-juriani** [Counselllor/Counsellor-Web#614]
<img width="348" alt="image" src="https://github.com/Counselllor/Counsellor-Web/assets/135598205/64ed3306-b069-4d73-871b-22d4b09cd4ed">

While creating an account, future date can be selected as well as date of birth.
We can allow the max date as of today (on which account is being created)

**Comment by mahak-juriani** [comment]
Request to please assign under gssoc, thank you

**Comment by thestarsahil** [comment]
Thanks for informing but you have raise 5 more major problem to contribute in this project which containing level 3

**Comment by mahak-juriani** [comment]
Hey @thestarsahil 
I didn't get it. Do you mean I have to raise 5 more major problems?

Also, I was done with this fix. Was just waiting for this issue to be assigned so that I can push code and create PR.
Please suggest what can I do here, thanks

**Comment by thestarsahil** [comment]
> Hey @thestarsahil I didn't get it. Do you mean I have to raise 5 more major problems?
> 
> Also, I was done with this fix. Was just waiting for this issue to be assigned so that I can push code and create PR. Please suggest what can I do here, thanks

Yes ! I meant to say this year we are accepting PR with repeatedly contributor contributing towards with major problem 
Not  just like raising only  single PR and then gone forever , instead we want contributor to work more often in one particular project and rasing more as contributor want !
That's oftern show the contribution and learning towards the open source 
Thank you

**Comment by mahak-juriani** [comment]
Hi @thestarsahil, 
I wanted to push the changes but it's failing with the message

My branch name was mahak-issue-614

Do i have to setup something on my github or will you allow some permission for me to be added as contributor
<img width="454" alt="image" src="https://github.com/Counselllor/Counsellor-Web/assets/135598205/1e2887fe-aad5-485a-8ef8-dc15ee23c160">

**Comment by mahak-juriani** [comment]
@thestarsahil please review this pull request #621
